caf-logging-serilog/src/main/docker/Dockerfile

```diff
@@ -0,0 +1,24 @@
+#
+# Copyright 2021 Micro Focus or one of its affiliates.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+FROM dockerhub-private.svsartifactory.swinfra.net/cafapi/opensuse-jdk8-maven:1.0.0
```

Comment:
- No open-source component can ever refer to internal company-private services.  (I know there are a few instances where this rule is broken but they are bugs.)
- Also we don't use `dockerhub-private` even internally anymore (except maybe for WebInspect).

Reply:
Removed